docs/40_development.md

```diff
@@ -86,6 +86,35 @@ controls Netlify settings. The @dandibot GitHub account is the "owner" of the
 Netlify account used for this purpose; in order to get access to that account,
 speak to @satra.
 
+## Monitoring
+
+### Service(s) status
+
+The DANDI project uses [upptime](https://upptime.js.org/) to monitor the status of DANDI provided and third-party services.
+The configuration is available in [.upptimerc.yml](https://github.com/dandi/upptime/blob/master/.upptimerc.yml) of the https://github.com/dandi/upptime repository, which is automatically updated by the upptime project pipelines.
+Upptime automatically opens new issues if any service becomes unresponsive, and closes issues whenever service comes back online.
+https://www.dandiarchive.org/upptime/ is the public dashboard for the status of DANDI services.
+
+## Logging
+
+### Sentry
+
+Sentry is used for error tracking main deployment.
+To access Sentry, login to https://dandiarchive.sentry.io .
+
+### Heroku & Papertrail
+
+`dandi-api` and `dandi-api-staging` apps have Papertrail add-on configured to capture logs.
+To access papertrail login to [Heroku dashboard](https://dashboard.heroku.com), proceed to the corresponding app and click on "Papertrail" add-on.
+
+CRON job on `drogon` server backups Papertrail logs as .csv files hourly at `/mnt/backup/dandi/papertrail-logs/{app}`.
+Moreover, `heroku logs` processes per app dump logs to `/mnt/backup/dandi/heroku-logs/{app}` directory.
```

Comment:
Unclear what this refers to. Another process on `drogon`?

Reply:
Right! Wording is indeed unclear although sounded fine for me originally ;-)  Is this making more sense?

```suggestion
Another CRON job on `drogon` runs `heroku logs` processes per app to dump logs to `/mnt/backup/dandi/heroku-logs/{app}` directory for each hour.
```